packages/vite/src/node/ssr/ssrManifestPlugin.ts

```diff
@@ -59,7 +59,7 @@ export function ssrManifestPlugin(config: ResolvedConfig): Plugin {
                   const chunk = bundle[filename] as OutputChunk | undefined
                   if (chunk) {
                     chunk.viteMetadata.importedCss.forEach((file) => {
-                      deps.push(`/${file}`)
+                      deps.push(join(config.base, file))
```

Comment:
I think this may break with relative base. We could merge it and then work on support for relative base though if you prefer, as this is already an improvement over the current situation.

Reply:
ok, merge first. and fix by `relative base`. I forgot we supported relative base 😂